hycom/RELO/src_2.2.98ZA-07Tsig0-i-sm-sse_relo_mpi/thermf.F

```diff
@@ -69,17 +70,20 @@ subroutine thermf_oi(m,n)
               surflx(i,j) = (1.0-covice(i,j))*surflx(i,j) +
      &                                        flxice(i,j)   !cell average
               sstflx(i,j) = (1.0-covice(i,j))*sstflx(i,j)   !relax over ocean
+              if (sss_underice==.false.) then
```

Comment:
If I read this correct.
When sst_underice=.false.
 Then before the sss relaxation was reduced by the ice concentration in points without rivers.
Now It is reduced by the ice concentration in all points.
If true then sss relaxation is never reduced due to the ice cover

Reply:
sst_underice has not been introduced because this model switches off the sst relaxation.

Reply:
So I understand you would like to comment sss_underice=.false. which meanings the sss relaxation was off (or reduced) due to ice cover.

Reply:
Let me rephrase: I think that SSS relaxation should be off under ice cover.

Reply:
> sst_underice has not been introduced because this model switches off the sst relaxation.
Sorry typo. sss_underice...
To be more precise. This reduces the relaxation by the percentage of ice cover. For this purpose it works. An alternative could be to say if icecon>0.15 (example) ssrlx=0.

Reply:
Personally I would prefer icecon>0.15 to the ice fraction in order to be more restrictive where the SSS climatology is bad. We may also have bad surprises when the model is run at lower resolution for longer periods.

Reply:
It will be a good constraint from my view also.